public/js/stylehelpers/freelancer.js

```diff
@@ -72,7 +73,7 @@ var animateObjects = function() {
       $hero = $(".parallax-hero.red");
   
   $parallax.css({
-    "background-position":"50% -"+ (( scrollTop / 4 ) + 500 )+ "px "
+    "background-position":"50% -"+ (( scrollTop / 4 ) + 500 )+ "px ",
```

Comment:
hmm. Interesting. I'd added the background-image here but then realized how bad of performance that would be so I put it in the stylesheet. I'll remove this, but this makes me want to look at the linting settings.